core/src/com/po/fuck/controller/movement/BasicMovement.java

```diff
@@ -1,7 +1,7 @@
-package com.po.fuck.movement;
+package com.po.fuck.controller.movement;
```

Comment:
I think movement should be in core, because it is not a controller, but a mechanic, and enemies also have it